vllm/worker/hpu_model_runner.py

```diff
@@ -2075,6 +2076,18 @@ def profile_run(self) -> None:
                              False, True)
         return
 
+    def _remove_duplicate_submodules_(self, model, inc_config):
```

Comment:
@kwisniewski98  Please only do it for deepseek models.

Reply:
Since it checks with mla_attn, I think current Codes is OK, no need to change to deepseek only

Reply:
Done